app/models/user.rb

```diff
@@ -691,6 +715,17 @@ def generic?
     !(admin? || teacher? || editor?)
   end
 
+  # for lectures that are older than one year, only the teacher or an editor
+  # of the course it belongs to can view users in forms related to the lecture
+  def can_view_users?(lecture)
+    return false unless can_edit?(lecture)
+    return true if can_edit?(lecture.course) || lecture.teacher == self
+    return true if lecture.course.term_independent
+    return true if !lecture.older_than?(1.year)
+
+    false
```

Comment:
might wanna use `return` here as well as it is used in the previous lines beforehand

Reply:
The ommision of return is  correct according to the [ruby style guide](https://rubystyle.guide/#no-explicit-return) und the standard rubocop settings. But I can change it, sure.

Reply:
Ah ok, I mean, if the ruby style guide says this is best practice, go for it ;) It's probably just me coming from other programming languages that I'm unused to this concept, but it should be fine, especially since this only affects the last line of a method.